src/Cronos/CronExpression.cs

```diff
@@ -41,36 +41,78 @@ public sealed class CronExpression: IEquatable<CronExpression>
         /// Equals to "0 0 1 1 *".
         /// </summary>
         public static readonly CronExpression Yearly = Parse("0 0 1 1 *", CronFormat.Standard);
+        
+        /// <summary>
+        /// Represents a cron expression that fires at an unspecified time once per year.
+        /// Equals to "H H H H H *".
+        /// </summary>
+        public static CronExpression YearlyWithJitter(Random rng) =>
```

Comment:
First of all, in my opinion your previous implementation with `int`-based seed was better in terms of safety and usability. `Random` instance is not a thread-safe data structure, and it's so easy to pass non-synchronized static instance here, that will cause issues. Also, constructing a `Random` instance is an additional step in terms of the API consumption that can be avoided.